src/httpserver/utilities/arguments.clj

```diff
@@ -0,0 +1,12 @@
+(ns httpserver.utilities.arguments
+  (:require [clojure.tools.cli :refer [parse-opts]]))
```

Comment:
Hm, missed this the first time through. I would recommend avoiding `refer` and just aliasing the namespace with `:as` (like you do in your tests). It's functionally the same here, but as a general rule of thumb you want to be consistent about how you're referring in functions from other namespaces and `:as` can't ever bite you, while `refer` could (what if you have a separate function in this namespace named `parse-opts`? Which fn are you calling?)